src/elements/calendar/calendar.snapshot.spec.ts

```diff
@@ -4,22 +4,30 @@ import { html } from 'lit/static-html.js';
 import { fixture, testA11yTreeSnapshot } from '../core/testing/private.js';
 
 import './calendar.js';
+import type { SbbCalendarElement } from './calendar.js';
 
 describe(`sbb-calendar`, () => {
-  it('renders', async () => {
-    const root = await fixture(
-      html`<sbb-calendar selected="2023-01-20T00:00:00" now="2023-01-04T00:00:00"></sbb-calendar>`,
-    );
+  describe('renders', () => {
+    let element: SbbCalendarElement;
+
+    beforeEach(async () => {
+      element = await fixture(html`
+        <sbb-calendar selected="2023-01-20T00:00:00" now="2023-01-04T00:00:00"></sbb-calendar>
+      `);
+    });
+
+    it('renders - DOM', async () => {
+      await expect(element).dom.to.be.equalSnapshot();
+    });
 
-    expect(root).dom.to.be.equal(
-      `<sbb-calendar now="2023-01-04T00:00:00" selected="2023-01-20T00:00:00"></sbb-calendar>`,
+    it('renders - Shadow DOM', async () => {
+      await expect(element).shadowDom.to.be.equalSnapshot();
+    });
+
+    testA11yTreeSnapshot(
+      html` <sbb-calendar now="2023-01-04T00:00:00" selected="2023-01-20T00:00:00"></sbb-calendar>`,
+      undefined,
+      { safari: true }, // We skip safari because it has an inconsistent behavior on ci environment
     );
```

Comment:
```suggestion
    testA11yTreeSnapshot(
      undefined,
      undefined,
      { safari: true }, // We skip safari because it has an inconsistent behavior on ci environment
    );
```
You can use the same template declared in the beforeEach